src/wp-includes/comment-template.php

```diff
@@ -2031,8 +2042,10 @@ function comment_id_fields( $post_id = 0 ) {
  *                                     being replied to.
  * @param bool         $link_to_parent Optional. Boolean to control making the author's name a link
  *                                     to their comment. Default true.
+ * @param int|WP_Post  $post           Optional. The post that the comment form is being displayed for.
+ *                                     Defaults to the current global post.
  */
-function comment_form_title( $no_reply_text = false, $reply_text = false, $link_to_parent = true ) {
+function comment_form_title( $no_reply_text = false, $reply_text = false, $link_to_parent = true, $post = null ) {
 	global $comment;
```

Comment:
@costdev The global is no longer assigned in this function. Should it be?

```php
// Sets the global so that template tags can be used in the comment form.
$comment = get_comment( $reply_to_id );
```

This line of code is now in `_get_comment_reply_id()`.